### ISSUE TITLE
Make 'rustlings' into a template repository

### ISSUE THREAD

**Issue by jakeisnt** [rust-lang/rustlings#682]
If this is a template repository, then users of rustlings, rather than forking, can simply click "Use this template" to solve the rustling exercises!

**Comment by HankB** [comment]
TL;DR - suggest a way to clone `rustlings` such that progress can be committed and pushed in order to facilitate continuing on different hosts.

I came here to suggest that the instructions recommend forking the project in order to be able to progress on different hosts (by committing work in progress to my own repo.) I don't know enough about Git/Github to suggest which approach is better. What I *do* know is that I cloned `rustlings`, got past the first quiz and then decided to continue on a different host. I proceeded to switch to a repo in my private Gitea server and in the process learned some new things about git. I could have saved some time by forking and cloning my repo where I presumably could have just committed/pushed my progress as I moved from host to host. I could have simply tarred and restored my copy of the `rustlings` directory but I think `git` is a better option to move from host to host.

Thanks!

**Comment by shadows-withal** [comment]
I think a lot of people already do this, I get misguided PRs from people committing their progress to their own forks on this repository all the time. The readme is already pretty packed, not sure if this'd be worth putting in there.

**Comment by mo8it** [comment]
The problem here would be avoided if go in the direction of https://github.com/rust-lang/rustlings/issues/1919 since Rustlings wouldn't be bound to git anymore.

**Comment by mo8it** [comment]
Closing in favor of https://github.com/rust-lang/rustlings/issues/1919
